lib/index.js

```diff
@@ -146,16 +147,16 @@ Appboy.prototype.initializeV2 = function(customEndpoint) {
   };
 
   if (customEndpoint) config.baseUrl = customEndpoint;
-  
+
   this.initializeTester(options.apiKey, config);
   window.appboy.initialize(options.apiKey, config);
-  
-  if (options.automaticallyDisplayMessages) window.appboy.display.automaticallyShowNewInAppMessages();
-  if (userId) window.appboy.changeUser(userId);
-  
-  window.appboy.openSession();
 
-  this.load('v2', this.ready);
+  if (options.automaticallyDisplayMessages) window.appboy.display.automaticallyShowNewInAppMessages();
+  if (userId) {
+    window.appboy.changeUser(userId, window.appboy.openSession(self.load('v2', self.ready)));
+  } else {
+    window.appboy.openSession(self.load('v2', self.ready));
```

Comment:
See note above -- this code passes `self.load(...)`'s return value to `openSession`. That seems like a mistake?